presto-accumulo/src/main/java/com/facebook/presto/accumulo/conf/AccumuloConfig.java

```diff
@@ -0,0 +1,237 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.accumulo.conf;
+
+import com.facebook.presto.accumulo.metadata.AccumuloMetadataManager;
+import com.facebook.presto.spi.PrestoException;
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.units.Duration;
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+import java.io.File;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+
+import static com.facebook.presto.accumulo.AccumuloErrorCode.VALIDATION;
+import static java.lang.String.format;
+
+/**
+ * File-based configuration properties for the Accumulo connector
+ */
+public class AccumuloConfig
+{
+    public static final String INSTANCE = "instance";
```

Comment:
All of these should probably start with "accumulo". For example, see `HiveClientConfig`